src/ab-test-preview.js

```diff
@@ -0,0 +1,58 @@
+import { getLetter } from '../src/utils';
+
+const queryString = window.location.search;
+const urlParams = new URLSearchParams(queryString).get( 'set_test' );
+const abtest = urlParams.get( 'set_test' ).replace( 'ab_block_', '' ).split( ':', 2 );
```

Comment:
@roborourke I am trying to use the way that you had originally suggested. I am fighting an issue for this when the using is using a preview URL for the page and not the AB Test URL I then have an issue around the `abtest` as the preview URL doesn't have a value for the `replace`.

Reply:
This is resolved through another method